cfgov/unprocessed/js/molecules/Notification.js

```diff
@@ -88,6 +88,42 @@ function Notification( element ) { // eslint-disable-line max-statements, inline
   }
 
   /**
+   * Generate a string of HTML from the plugin's settings
+   * @returns {string} The expanded HTML string
+   * Commented out until nedded.
+
+  function generateHTML( settings ) {
```

Comment:
yeah, removed in https://github.com/cfpb/cfgov-refresh/pull/2482/commits/ad79459ed120111d3e457e2ce40334acb2f96343